### ISSUE TITLE
How do I change post title from Django administration?

### ISSUE THREAD

**Issue by brilam** [jazzband/django-tinymce#353]
Hi,

I am pretty new to django-tinymce and I am not sure I've missed the documentation on this. I've gotten it setup and have created a Post object. Upon saving these posts, I see this in my Django admin.

![image](https://user-images.githubusercontent.com/19829358/117601947-7e88e200-b11d-11eb-8863-2dcb40061338.png)

However, I don't want it to say Post object (3). Instead, I want it to say the Post's title:

![image](https://user-images.githubusercontent.com/19829358/117601973-8ba5d100-b11d-11eb-94df-4516b574318f.png)

So in other words, I want Post object (3) here to say A Title. How would I do this?

**Comment by brilam** [comment]
Well, that was a silly question. For anyone wondering, just add __str__ to your model.
